src/Microsoft.Azure.SignalR.Common/Utilities/AuthenticationHelper.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Azure.SignalR
 {
     internal static class AuthenticationHelper
     {
+        // Current limitaion is in Nigix for 4K
```

Comment:
```suggestion
         // Current limitation is in Nginx for 4K
```

Reply:
remove the internal implementation  details from the comment, as we might change that and there is no need to expose that